import-service/src/importFileParser.ts

```diff
@@ -0,0 +1,67 @@
+import { S3Event, S3Handler } from 'aws-lambda'
+import * as AWS from 'aws-sdk'
+import * as csv from 'csv-parser'
+import { BUCKET } from './bucketOptions'
+import 'source-map-support/register'
+
+export const importFileParser: S3Handler = async (event: S3Event) => {
+  console.log('lambda : importFileParser, event: ', event)
+
+  const s3 = new AWS.S3({ region: 'eu-west-1' })
```

Comment:
Maybe wrap all of this in `try catch`?

Reply:
Done. [3077edc](https://github.com/YakovGrischenko/nodejs-aws-be/pull/3/commits/3077edc16731328be56cf30709bf6943197e75a4)